pages/learn/manage/account.md

```diff
@@ -174,6 +174,12 @@ If you wish to delete your {{ $names.cloud.lower }} account, go to your [_Prefer
 [ssh]:/learn/manage/ssh-access
 [starter]:/learn/manage/app-types#starter
 [administrator]: /learn/manage/organizations/#managing-roles--access-in-an-organization
+[device-actions]:/learn/manage/actions/
+[env-variables]:/learn/manage/serv-vars/
+[fleet-variables]:/learn/manage/configuration/
+[ssh-access]:/learn/manage/ssh-access/
```

Comment:
This is the same as the `[ssh]` link just a few lines above?

Reply:
Thanks @lurch I must have missed that one out. Will remove this and use SSH reference link only.